File: reactapp/src/models/HR/InitAppData.ts
import { defaultPositionDTO, PositionDTO } from "@/pages/Positions/models/Position";
import { DepartmentDTO, defaultDepartmentDTO } from "../../pages/Departments/models/Department";

export interface InitAppData {
    positionInsert: PositionDTO,
    departmentInsert: DepartmentDTO
}

export const defaultInitAppData: InitAppData = {
    positionInsert: defaultPositionDTO,
    departmentInsert: defaultDepartmentDTO,
}

File: webapi/Models/InitAppData.cs
using webapi.Models.HR;

namespace webapi.Models
{
    public record InitAppData
    {
        public required PositionDTO PositionInsert {  get; set; }
        public required DepartmentDTO DepartmentInsert { get; set; }
        public required DepartmentTeamInsert DepartmentTeamInsert { get; set; }
    }
}
